proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,456 @@
+project Kanban
+board](https://docs.github.com/en/issues/organizing-your-work-with-project-boards).
+This project board will track GitHub issues from the contribution-ready stage
+through completion by merging a pull request. Each lane of the project board
+maps to a stage of the contribution process. New and existing GitHub labels and
+GitHub issue events will be used to drive the movement of the issue between the
+lanes of the GitHub project, which map to the stages of the contribution
+process. GitHub issue labels will be used to move an issue to the next lane.
+
+The contribution-ready lane is the entry point for users interested in making a
+contribution. The ready for review lane is the prioritization queue, and issues
+will be ordered in decreasing priority from top to bottom.
+
+
+## Managing the existing backlog
+
+As of 2021-12-15, there are 188 pull requests and 436 issues in the AWS CLI
+GitHub repository. 43 pull requests and 87 issues have not been updated in more
+than a year. 95 issues have ten or more upvotes. 
+<!-- [TBD] open pull requests are directly related to an existing feature request. -->
+
+Based on these numbers, we propose to manage existing pull requests in two
+phases. First, we will determine if there is a corresponding tracking issue for
+each pull request and [link them
+together](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
+If no tracking issue exists, we will create one and link it to the pull request.
+Then, we will review all open issues and apply the new specification to identify
+the current stage. We will prioritize the review of issues with a linked pull
+request. After review, all pull requests will follow the process and timelines
+defined in the specification.
+
+## Rationale/FAQ
+
+### Q. Why do we limit the number of in flight reviews?
+
+We do not want to discourage users from contributing or proposing changes, but
+we need to acknowledge that the maintainers have limited bandwidth to give
+feedback on implementation or design and review pull requests. The maintainers
+have work that comes from outside of the GitHub repository that needs to be
+prioritized along with user contributions. Controlling the volume of pull
+request reviews and prioritizing completing the in flight reviews over new
+reviews will result in a shorter time to merge and release a contributor's
+change for a feature requested by the community. It will also reduce the amount
+of context switching for the maintainers, allowing them to focus on giving high
+quality feedback to users. As we use this framework to process existing feature
+requests, we can revisit what the in flight limit should be.
+
+### Q. Why do we require issues to be opened instead of just pull requests?
+
+Having a defined process for intake reduces the burden on deciding which way is
+the best way. We're taking an opinionated stance that code changes made in
```

Comment:
For this first sentence, the wording feels awkward/vague. How about this instead?

It removes any ambiguity on how to initiate a contribution by ensuring that all potential contributions must start out as a GitHub issue.